services/app-api/utils/validation/completionSchemas.ts

```diff
@@ -16,6 +16,8 @@ export const error = {
   INVALID_URL: "Response must be a valid hyperlink/URL",
   INVALID_DATE: "Response must be a valid date",
   INVALID_END_DATE: "End date can't be before start date",
+  NUMBER_LESS_THAN_ZERO: "Response must be equal to or greater than zero",
+  NUMBER_LESS_THAN_ONE: "Response must be equal to or greater than one",
```

Comment:
Nit: This wording sounds weird to me. Should it be "Response must be greater than or equal to x" instead?